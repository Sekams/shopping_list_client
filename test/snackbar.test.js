import React from 'react';
import SnackBar from '../src/components/snackbar'

test("renders the snackbar component", () => {
    const wrapper = shallow(
        <SnackBar />
    );
    expect(wrapper).toMatchSnapshot();
});