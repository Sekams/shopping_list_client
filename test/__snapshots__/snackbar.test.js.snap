// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the snackbar component 1`] = `
<div
  id="snackbar"
>
  <p />
</div>
`;
